e2e_tests/integration/topics.spec.ts

```diff
@@ -31,9 +31,7 @@ describe('Help topics', () => {
     cy.executeCommand(':clear')
     const query = ':help commands'
     cy.executeCommand(query)
-    cy.get('[data-testid="frameCommand"]', { timeout: 10000 })
-      .first()
-      .should('contain', query)
+    cy.get('[data-testid="frameCommand"]', { timeout: 10000 }).contains(query)
```

Comment:
Not sure I like these `.should('contain', x)` -> `contains(x)` changes. 
The former is an explicit assertion where the second returns the element so we can chain other operations on it.
Both will fail if the element isn't there, but the explicit assertion is a more explicit failure if I remember correctly.

Reply:
I agree the should contain is clearer as an assertion. The reason I changed them as because the dom structure was changed and .contains() looks through all grandchildren rather than just its own. So I use .contains() as it accomplishes the same thing but was easier to write. I could change them back if you'd like

Reply:
I didn't know that `.contains` checks deeper. No need to change then.